src/Stryker.Core/Stryker.Core/Reporters/Progress/ProgressReporter.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Stryker.Core.Mutants;
+using System;
+
+namespace Stryker.Core.Reporters.Progress
+{
+    public interface IProgressReporter
+    {
+        void ReportRunTest(TimeSpan testDuration, IReadOnlyMutant mutantTestResult);
+        void ReportInitialState(int totalNumberOfTests);
+    }
+
+    public class ProgressReporter : IProgressReporter
```

Comment:
Why did you create a new interface for this reporter? The current IReporter interface would also work here, making switching reporters at runtime possible.